app/assets/javascripts/transactions.js

```diff
@@ -8,6 +8,12 @@ $(function () {
 
   showHidePeriodAdditionalInput();
 
+  if ($('#flow').length) {
+    $('body').css('padding-bottom', '130px');
+  } else {
+    $('body').css('padding-bottom', '45px');
```

Comment:
@yunixon it will be better ti add some class to body tag on that pages, instead of adding this hack in js.